proto/umee/leverage/v1/events.proto

```diff
@@ -0,0 +1,108 @@
+syntax = "proto3";
+package umee.leverage.v1;
+
+import "cosmos/base/v1beta1/coin.proto";
+import "cosmos_proto/cosmos.proto";
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/umee-network/umee/v3/x/leverage/types";
+
+option (gogoproto.goproto_getters_all) = false;
+
+// EventSupply is emitted on Msg/Supply
+message EventSupply {
+  // Liquidity supplier bech32 address.
+  string supplier = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // Liquidity provided to the module.
+  cosmos.base.v1beta1.Coin supplied = 2 [(gogoproto.nullable) = false];
+  // uAsset received by the supplier in exchange for the
+  // provided liquidity.
+  cosmos.base.v1beta1.Coin uasset = 3 [(gogoproto.nullable) = false];
+}
+
+// EventWithdraw is emitted on Msg/Withdraw
+message EventWithdraw {
+  // Liquidity supplier bech32 address.
+  string supplier = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // uAsset sent to the module in exchange for the underlying asset.
+  cosmos.base.v1beta1.Coin uasset = 2 [(gogoproto.nullable) = false];
+  // Liquidity received by the supplier.
+  cosmos.base.v1beta1.Coin asset = 3 [(gogoproto.nullable) = false];
+}
+
+// EventCollaterize is emitted on Msg/Collaterize
+message EventCollaterize {
+  // Borrower bech32 address.
+  string borrower = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // uAsset provided as a collateral.
```

Comment:
```suggestion
  // uTokens provided as a collateral.
```

Reply:
Don't we use `Asset` in proto?

Reply:
Asset yes, but `uAsset` isn't used anywhere.

Reply:
OK, will update to `uToken` since it is already used in other parts of the code. `uAsset` sounds just more logical to me.

Reply:
Especially this part looks weird `Utoken:   msg.Asset`